testing/coreruleset/go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/corazawaf/coraza/v3/testing/coreruleset
 
 go 1.22.3
```

Comment:
```suggestion
go 1.22.3
```
In other places you moved to `go 1.22.0`, should we be consistent also here?

Reply:
unfortunately we cannot https://github.com/coreruleset/albedo/blob/v0.0.16/go.mod#L3

Reply:
ouch, albedo. thanks!